roles/nextflow/tasks/main.yml

```diff
@@ -18,6 +18,7 @@
   get_url:
     url: "{{ nextflow_download_url }}"
     dest: "{{ nextflow_dest }}/nextflow"
+    mode: u=rwx
```

Comment:
Yes and no.. The explicit setting of mode here is because the deployment script actually invokes Nextflow as a task (I believe for some initialization stuff). At the end of the deployment, the script will recursively set all permissions, so before this ends up on the cluster, the permissions should have been set so that the users can access regardless.

Reply:
Ok, good to know!